cli/silence_add.go

```diff
@@ -56,8 +56,16 @@ var addCmd = &cobra.Command{
 }
 
 func init() {
-	user, _ := user.Current()
-	addCmd.Flags().StringP("author", "a", user.Username, "Username for CreatedBy field")
+	var username string
+
+	user, err := user.Current()
+	if err != nil {
+		fmt.Printf("Failed to get the current user, specify one with --author: %s\n", err)
```

Comment:
It's typical in go to have the first letter of a log or print statement be lowercase, so `failed` instead of `Failed`. Also, err is of type `error`, which is printed out with `%v`. Once that's done, good to go.